database/schema.sql

```diff
@@ -0,0 +1,62 @@
+CREATE DATABASE catalog_of_things;
+
+CREATE TABLE item(
+    id SERIAL PRIMARY KEY,
+    genre VARCHAR(150),
+    author VARCHAR(150),
+    source VARCHAR(150),
+    label VARCHAR(150),
```

Comment:
- [ ] Kindly update these properties to reference their respective classes, as they represent the foreign keys of the classes

**Example:** This is only for the `genre` table, but you have to apply the same thing for the rest of the table (author, source label)
```rb
...
genre_id INTEGER, FOREIGN KEY(genre_id) REFERENCES genre(id)
...
```

Reply:
@LionRouge1 Hello thank you for your nice review, it mans instead of this we have to like this:
```
CREATE TABLE item (
    id SERIAL PRIMARY KEY,
    genre_id INTEGER,
    author_id INTEGER,
    source_id INTEGER,
    label_id INTEGER,
    publish_date DATE,
    archived BOOLEAN,
    FOREIGN KEY (genre_id) REFERENCES genre (id),
    FOREIGN KEY (author_id) REFERENCES author (id),
    FOREIGN KEY (source_id) REFERENCES source (id),
    FOREIGN KEY (label_id) REFERENCES label (id)
);
```